services/web/server/src/simcore_service_webserver/director_v2_settings.py

```diff
@@ -25,6 +26,62 @@ class DirectorV2Settings(BaseCustomSettings, MixinServiceSettings):
     def base_url(self) -> URL:
         return URL(self._build_api_base_url(prefix="DIRECTOR_V2"))
 
+    # DESIGN NOTE:
+    # - Timeouts are typically used in clients (total/read/connection timeouts) or asyncio calls
+    # - Mostly in floats (aiohttp.Client/) but sometimes in ints
+    # - Typically in seconds but occasionally in ms
+    DIRECTOR_V2_STOP_SERVICE_TIMEOUT: PositiveInt = Field(
+        _HOUR + 10,
+        description=(
+            "Timeout on stop service request (seconds)"
+            "ANE: The below will try to help explaining what is happening: "
+            "webserver -(stop_service)-> director-v* -(save_state)-> service_x"
+            "- webserver requests stop_service and uses a 01:00:10 timeout"
+            "- director-v* requests save_state and uses a 01:00:00 timeout"
+            "The +10 seconds is used to make sure the director replies"
+        ),
+        envs=[
+            "DIRECTOR_V2_STOP_SERVICE_TIMEOUT",
+            # TODO: below this line are deprecated. rm when deveops give OK
+            "WEBSERVER_DIRECTOR_STOP_SERVICE_TIMEOUT",
+            "webserver_director_stop_service_timeout",
+        ],
+    )
+
+    DIRECTOR_V2_RESTART_DYNAMIC_SERVICE_TIMEOUT: PositiveInt = Field(
+        1 * _MINUTE,
+        description="timeout of containers restart",
+        envs=[
+            "DIRECTOR_V2_RESTART_DYNAMIC_SERVICE_TIMEOUT",
+            # TODO: below this line are deprecated. rm when deveops give OK
+            "SERVICES_COMMON_RESTART_CONTAINERS_TIMEOUT",
+            "SERVICES_COMMON_restart_containers_timeout",
+        ],
+    )
+
+    DIRECTOR_V2_STORAGE_SERVICE_UPLOAD_DOWNLOAD_TIMEOUT: PositiveInt = Field(
+        _HOUR,
+        description=(
+            "When dynamic services upload and download data from storage, "
+            "sometimes very big payloads are involved. In order to handle "
+            "such payloads it is required to have long timeouts which "
+            "allow the service to finish the operation."
+        ),
+        envs=[
+            "DIRECTOR_V2_DYNAMIC_SERVICE_DATA_UPLOAD_DOWNLOAD_TIMEOUT",
+            # TODO: below this line are deprecated. rm when deveops give OK
+            "SERVICES_COMMON_STORAGE_SERVICE_UPLOAD_DOWNLOAD_TIMEOUT",
+            "SERVICES_COMMON_storage_service_upload_download_timeout",
+        ],
+    )
+
+    def get_service_retrieve_timeout(self) -> ClientTimeout:
```

Comment:
why is this one the only "special case"? all other timeouts do not have a constructor here.